modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -934,23 +936,25 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 
 		_assertCount(4);
 
+		HashMap<String, Serializable> invalidValues =
```

Comment:
I believe we can rename to `values` only. What do you thing? i don't see any variable with `invalidValues` in tests...